tests/integration/test_wpk/test_wpk_manager.py

```diff
@@ -0,0 +1,844 @@
+            'sha_list' : ['NOT_NEED'],
+            'upgrade_exec_result' : ['0'],
+            'upgrade_script_result' : [0],
+            'status': ['In progress'],
+            'upgrade_notification': [False],
+            'message_params': {'file_path' : 'wpk_test.wpk', 'installer' : 'custom_installer.sh'},
+            'error_msg' : ['Not need'],
+            'checks' : ['wpk_name'],
+            'expected_response' : 'Success',
+            'command' : 'upgrade_custom'
+        }
+    },
+    # 19. Single Agent with use_http = 1 - success
+    {
+        'params': {
+            'PROTOCOL': 'tcp',
+            'WPK_REPOSITORY' : WPK_REPOSITORY_4x,
+            'CHUNK_SIZE' : CHUNK_SIZE,
+            'TASK_TIMEOUT' : TASK_TIMEOUT
+        },
+        'metadata' : {
+            'wpk_repository' : WPK_REPOSITORY_4x,
+            'agents_number': 1,
+            'protocol': 'tcp',
+            'agents_os': ['debian7'],
+            'disconnect' : [False],
+            'sha_list' : ['VALIDSHA1'],
+            'upgrade_exec_result' : ['0'],
+            'upgrade_script_result' : [0],
+            'status': ['Done'],
+            'upgrade_notification': [True],
+            'message_params': {'use_http' : 1},
+            'checks' : ['use_http', 'version'],
+            'expected_response' : 'Success'
+        }
+    },
+    # 20. Single Agent with use_http = default - success
+    {
+        'params': {
+            'PROTOCOL': 'tcp',
+            'WPK_REPOSITORY' : WPK_REPOSITORY_4x,
+            'CHUNK_SIZE' : CHUNK_SIZE,
+            'TASK_TIMEOUT' : TASK_TIMEOUT
+        },
+        'metadata' : {
+            'wpk_repository' : WPK_REPOSITORY_4x,
+            'agents_number': 1,
+            'protocol': 'tcp',
+            'agents_os': ['debian7'],
+            'disconnect' : [False],
+            'sha_list' : ['VALIDSHA1'],
+            'upgrade_exec_result' : ['0'],
+            'upgrade_script_result' : [0],
+            'status': ['Done'],
+            'upgrade_notification': [True],
+            'checks' : ['use_http', 'version'],
+            'expected_response' : 'Success'
+        }
+    },
+    # 21. Single Agent with use_http = 0 - success
+    {
+        'params': {
+            'PROTOCOL': 'tcp',
+            'WPK_REPOSITORY' : WPK_REPOSITORY_4x,
+            'CHUNK_SIZE' : CHUNK_SIZE,
+            'TASK_TIMEOUT' : TASK_TIMEOUT
+        },
+        'metadata' : {
+            'wpk_repository' : WPK_REPOSITORY_4x,
+            'agents_number': 1,
+            'protocol': 'tcp',
+            'agents_os': ['debian7'],
+            'disconnect' : [False],
+            'sha_list' : ['VALIDSHA1'],
+            'upgrade_exec_result' : ['0'],
+            'upgrade_script_result' : [0],
+            'status': ['Done'],
+            'message_params': {'use_http' : 1},
+            'upgrade_notification': [True],
+            'checks' : ['use_http', 'version'],
+            'expected_response' : 'Success'
+        }
+    }
+]
```

Comment:
Remove the white space between the key and the colon. Check that there is a comma and a white space between the different elements of a list